docker/development/debian/Dockerfile

```diff
@@ -44,17 +44,19 @@ RUN apt-get update -y \
  && rm -rf /var/lib/apt/lists/*
 
 ## Python installing dependencies
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Should be `ENV` no?

Reply:
This is just for the build step so that it doesn't crash when trying to find the timezone. Env would be if we want it around for future runs?

Reply:
Woops, fair!